documentation/objects/objects.md

```diff
@@ -17,49 +17,46 @@ weight: 1
-feature to run object classification on much larger images (prediction only -- no training.)
+**Size Limitation:**
 
-<a href="figs/ilastik_start_screen.png" data-toggle="lightbox"><img src="figs/ilastik_start_screen.png" class="img-responsive" /></a>
-
-### Pixel Classification + Object Classification
-This is a combined workflow, which lets you start from the raw data, perform pixel classification as described 
-in the
-[Pixel Classification workflow docs]({{site.baseurl}}/documentation/pixelclassification/pixelclassification.html)
-and then thresholding the probability maps to obtain a segmentation that you then use in Object Classification.
-This workflow is primarily meant for demo purposes.
-For serious projects, we recommend to use the two workflows, [Pixel Classification]({{site.baseurl}}/documentation/pixelclassification/pixelclassification.html) and Object Classification separately using the generated output form the former as an additional input in the latter one.
+For object classification, images used in _training_ have to be small enough to fit entirely into your machine's RAM.
+However, once you have created a satisfactory classifier using one or more small images (or cutouts from your complete dataset), you can use the "Blockwise Object Classification"
+feature to run object classification on much larger images (prediction only - no training).
 
-<a href="figs/input_pixel_class.png" data-toggle="lightbox"><img src="figs/input_pixel_class.png" class="img-responsive" /></a>
+<a href="figs/ilastik_start_screen.png" data-toggle="lightbox"><img src="figs/ilastik_start_screen.png" class="img-responsive" /></a>
 
 ### Object Classification [Inputs: Raw Data, Pixel Prediction Map]
-You should choose this workflow if you have pre-computed probability
-maps.
-The data input applet of this workflow expects you to load the probability maps in addition to the raw data:
+You should choose this workflow if you have a probability map, i.e. each pixel value is the pixel's probability to belong to an object.
+To obtain a segmentation from this input, the workflow includes a step for thresholding the probabilities.
+If you use the Pixel Classification workflow to identify objects, we recommend you export the probability maps there and then continue with this object classification workflow.
+
+Load the probability maps in addition to the raw data in the Input Data step:
 
 <a href="figs/input_prediction_image.png" data-toggle="lightbox"><img src="figs/input_prediction_image.png" class="img-responsive" /></a>
 
 ### Object Classification [Inputs: Raw Data, Segmentation]
-This workflow should be used if you already have a binary segmentation image. 
+This workflow should be used if you already have a binary segmentation image, or an object identity map (object labels).
+You could use a "Simple segmentation" export from the Pixel Classification workflow if this is sufficient for your use case, and thereby skip the thresholding step mentioned above.
```

Comment:
it doesn't work with the simple segmenation - it has pixel values starting from 1...n_classes. Object classification requires the background to have the value 0.